src/test/scala-2.+/com/fasterxml/jackson/module/scala/util/ScalaObjectTest.scala

```diff
@@ -0,0 +1,53 @@
+package com.fasterxml.jackson.module.scala.util
+
+import org.scalatest.matchers.should.Matchers.{contain, convertToAnyShouldWrapper, empty}
+import org.scalatest.wordspec.AnyWordSpecLike
+
+object TestObject
+
+case object TestCaseObject
+
+class TestClass
```

Comment:
why not add to `src/test/scala` ? does this not work with scala 3?

Reply:
I think these are not necessary in tests but they are used just to make sure ScalaObject.unapply do not detect classes with names that end with a `$` as scala singleton objects

Reply:
These class names are probably legal in scala 2 but illegal in scala 3. Since I haven't seen anyone in their right mind to use such class names I'm deleting them.